packages/react/src/core/packs.ts

```diff
@@ -58,6 +58,13 @@ export const print = {
 			height: 'auto !important',
 		},
 	},
+	// for context that requires background on print (eg. white text on dark background)
+	exactColor: {
+		'@media print': {
+			'-webkitPrintColorAdjust': 'exact',
```

Comment:
I think you either do this:
```suggestion
			WebkitPrintColorAdjust: 'exact',
```

or this:

```suggestion
			'-webkit-print-color-adjust': 'exact',
```

Reply:
`WebkitPrintColorAdjust` was giving TS warnings but `'-webkit-print-color-adjust' `is good. Swapped to `'-webkit-print-color-adjust'`

[Updated in 0379ffa](https://github.com/agriculturegovau/agds-next/pull/1965/commits/0379ffa79d964d05e6a656e44110ff4ace666c90#diff-f2a0f4739efb990f5076919ca5c0982b1ba7f5432ddd52a08e2f74ff8e22bcc0)